cmake/OpenTurbineOptions.cmake

```diff
@@ -16,7 +16,8 @@ macro(openturbine_setup_options)
   option(OpenTurbine_ENABLE_CPPCHECK "Enable CppCheck analysis" OFF)
   option(OpenTurbine_ENABLE_PCH "Enable precompiled headers" OFF)
   option(OpenTurbine_ENABLE_VTK "Use VTK for visualization" OFF)
-  option(OpenTurbine_BUILD_OPENFAST_ADI "Build the OpenFAST ADI external project" ON)
+  option(OpenTurbine_BUILD_OPENFAST_ADI "Build the OpenFAST ADI external project" OFF)
+  option(OpenTurbine_BUILD_ROSCO_CONTROLLER "Build the ROSCO controller external project" ON)
```

Comment:
I'm thinking anything that builds an external project should be off by default.  What do you think?

Reply:
I agree. I'll change it to OFF by default